conans/client/conf/__init__.py

```diff
@@ -83,6 +83,7 @@
 default_profile = %s
 compression_level = 9                 # environment CONAN_COMPRESSION_LEVEL
 sysrequires_sudo = True               # environment CONAN_SYSREQUIRES_SUDO
+# vs_installation_preference = Enterprise, Professional, Community, BuildTools # environment CONAN_VS_INSTALLATION_PREFERENCE
```

Comment:
This is buggy, as ``get_env()`` will parse the list as ``["Enterprise", " Professional",...`` with extra spaces

Reply:
strip now done in ``get_env()``